client/sites-dashboard/components/sites-table-row/docs/example.jsx

```diff
@@ -0,0 +1,212 @@
+			product_name_short: 'Free',
+			expired: false,
+			user_is_owner: false,
+			is_free: true,
+		},
+		is_wpcom_atomic: false,
+		title: 'The long title of the sample site',
+		slug: 'wpvip.wordpress.com',
+	};
+
+	const siteSimpleFree = {
+		...exampleSite,
+		ID: 1,
+	};
+
+	const siteSimpleBusinessComingSoon = {
+		...exampleSite,
+		ID: 2,
+		options: {
+			...exampleSite.options,
+			updated_at: new Date( +new Date() - 60 * 60 * 24 * 1000 ),
+		},
+		plan: {
+			...exampleSite.plan,
+			product_id: 1008,
+			product_slug: 'business-bundle',
+			product_name_short: 'Business',
+			user_is_owner: true,
+			is_free: false,
+		},
+		is_coming_soon: true,
+		launch_status: 'unlaunched',
+	};
+
+	const siteSimpleBusinessExpired = {
+		...exampleSite,
+		ID: 3,
+		options: {
+			...exampleSite.options,
+			updated_at: new Date( +new Date() - 60 * 60 * 24 * 1000 ),
+		},
+		plan: {
+			...exampleSite.plan,
+			product_id: 1008,
+			product_slug: 'business-bundle',
+			product_name_short: 'Business',
+			user_is_owner: true,
+			is_free: false,
+			expired: true,
+		},
+	};
+
+	const siteJetpackComplete = {
+		...exampleSite,
+		ID: 4,
+		jetpack: true,
+		options: {
+			...exampleSite.options,
+			updated_at: new Date( +new Date() - 60 * 60 * 24 * 2 * 1000 ),
+		},
+		plan: {
+			...exampleSite.plan,
+			product_id: 2014,
+			product_slug: 'jetpack_complete',
+			product_name_short: 'Complete',
+			user_is_owner: true,
+			is_free: false,
+		},
+	};
+
+	const siteSimplePrivateP2 = {
+		...exampleSite,
+		ID: 5,
+		is_private: true,
+		visible: false,
+		options: {
+			...exampleSite.options,
+			is_wpforteams_site: true,
+			updated_at: new Date( +new Date() - 60 * 60 * 24 * 30 * 1000 ),
+		},
+		plan: {
+			...exampleSite.plan,
+			product_id: 1040,
+			product_slug: '"wp_p2_plus_monthly"',
+			product_name_short: 'P2+',
+			is_free: false,
+		},
+		icon: {
+			ico: 'https://wpjobmanager.com/wp-content/uploads/2022/10/wp-job-manager-logo.webp?w=16',
+			img: 'https://wpjobmanager.com/wp-content/uploads/2022/10/wp-job-manager-logo.webp',
+		},
+		p2_thumbnail_elements: {
+			color_link: '#0267ff',
+			color_sidebar_background: '#d0ccb8',
+			header_image:
+				'https://wpjobmanager.com/wp-content/uploads/2022/11/hero-resized.png?resize=680',
+		},
+	};
+
+	return (
+		<div>
+			<h2>Free Simple Site</h2>
+			<div
+				style={ {
+					backgroundColor: 'white',
+					border: '1px solid lightgray',
+					padding: '16px',
+				} }
+			>
+				<table>
+					<tbody>
+						<SitesTableRow site={ siteSimpleFree } key={ siteSimpleFree.ID }></SitesTableRow>
+					</tbody>
+				</table>
+			</div>
+			<h2>Business Simple Site with Coming Soon status</h2>
+			<div
+				style={ {
+					backgroundColor: 'white',
+					border: '1px solid lightgray',
+					padding: '16px',
+				} }
+			>
+				<table>
+					<tbody>
+						<SitesTableRow
+							site={ siteSimpleBusinessComingSoon }
+							key={ siteSimpleBusinessComingSoon.ID }
+						></SitesTableRow>
```

Comment:
Because we are not passing children to the component, we can close the tag earlier.
```suggestion
						<SitesTableRow
							site={ siteSimpleBusinessComingSoon }
							key={ siteSimpleBusinessComingSoon.ID }
						/>
```

Reply:
Thanks, changed to short form.